config/external_db_map/default.txt

```diff
@@ -133,11 +139,16 @@ GenBank	GenBank_translation	translation
 GB	GenBank_translation	translation
 GenBank_translation	GenBank_translation	translation
 
+HAMAP	HAMAP	translation
+
 INTERPRO	Interpro	translation
 
 MEROPS	MEROPS	translation
 
 PDB	PDB	translation
+PFAM	PFAM	translation
```

Comment:
Just wondering why such mapping only added now ? Would of expected PFAM in place

Reply:
PFAM is in place, but we just allowing it to be attached only to the peptide/CDS/translation xrefs